### ISSUE TITLE
Errors in Intellij themes

### ISSUE THREAD

**Issue by AlexFalappa** [JFormDesigner/FlatLaf#26]
While playing with the demo application and choosing some Intellij Themes the following errors were print on the terminal:
```
Failed to parse: 'MenuBar.borderColor=##1a1721'
    invalid color '##1a1721'
Failed to parse: 'CheckBoxMenuItem.disabledBackground=#darkBackground'
    invalid color '#darkBackground'
Failed to parse: 'CompletionPopup.matchForeground=#42A5F52'
    invalid color '#42A5F52'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#42A5F52'
    invalid color '#42A5F52'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#2979ff2'
    invalid color '#2979ff2'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#2979ff2'
    invalid color '#2979ff2'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#2979ff2'
    invalid color '#2979ff2'
Failed to parse: 'List.selectionInactiveBackground=#DBDBDC8025'
    invalid color '#DBDBDC8025'
Failed to parse: 'Table.selectionInactiveBackground=#DBDBDC8025'
    invalid color '#DBDBDC8025'
Failed to parse: 'Tree.selectionInactiveBackground=#DBDBDC8025'
    invalid color '#DBDBDC8025'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#2979ff2'
    invalid color '#2979ff2'
Failed to parse: 'List.selectionInactiveBackground=#DBDBDC8025'
    invalid color '#DBDBDC8025'
Failed to parse: 'Table.selectionInactiveBackground=#DBDBDC8025'
    invalid color '#DBDBDC8025'
Failed to parse: 'Tree.selectionInactiveBackground=#DBDBDC8025'
    invalid color '#DBDBDC8025'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#FF79C52'
    invalid color '#FF79C52'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#FF79C52'
    invalid color '#FF79C52'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#79CB602'
    invalid color '#79CB602'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#79CB602'
    invalid color '#79CB602'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#2693862'
    invalid color '#2693862'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
Failed to parse: 'CompletionPopup.matchForeground=#d336822'
    invalid color '#d336822'
Failed to parse: 'Viewport.foreground=#%fc'
    invalid color '#%fc'
```

It seems the included `*.theme.json` files contain wrong color definitions or missing color references.

I understand that maybe these should be reported to the original authors of the themes.

**Comment by DevCharly** [comment]
Yes, this are bugs in the 3rd party themes.
IntelliJ IDEA silently ignores such invalid colors.

I've now reported the issues:
- https://github.com/OlyaB/DarkPurpleTheme/issues/20
- https://github.com/Hiberbee/code-highlight-themes/issues/13
- https://github.com/mallowigi/material-theme-ui-lite/issues/5

We'll see whether they get fixed.

I don't want to fix this issues in the demo's `.theme.json` files because there are so many files affected and then updating the themes (with [IJThemesUpdater](https://github.com/JFormDesigner/FlatLaf/blob/master/flatlaf-demo/src/main/java/com/formdev/flatlaf/demo/intellijthemes/IJThemesUpdater.java)) becomes a pain...

**Comment by AlexFalappa** [comment]
@DevCharly I understand your point.

Feel free to close this issue if you think it's not valid.

**Comment by DevCharly** [comment]
No, this is a valid issue and I'd like to keep it open until all themes are fixed.

I think there is a good chance that the material themes get fixed because the author has already replied to the report. Then 98% of the invalid colors should be gone.

`##1a1721` in DarkPurple theme is accepted as valid color by IntelliJ IDEA because there are two places where the leading '#' is removed. So I will fix this in FlatLaf.

Remains the invalid color in Hiberbee theme...

**Comment by DevCharly** [comment]
all invalid colors are now fixed in 0.24
